support-frontend/Brewfile

```diff
@@ -1,2 +1,6 @@
 tap "guardian/devtools"
 brew "guardian/devtools/dev-nginx"
+brew "nginx"
```

Comment:
nginx will be [pulled in by `dev-nginx`](https://github.com/guardian/homebrew-devtools/blob/master/Formula/dev-nginx.rb#L8)